lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -133,6 +139,146 @@ def test_create_thread(self, mock_request):
         )
         assert_equal(response.status_code, 200)
 
+
+    @patch.dict("django.conf.settings.FEATURES", {"ENABLE_NOTIFICATIONS": True})
+    def test_create_cohorted_thread(self, mock_request):
```

Comment:
This is a really big test that appears to be validating multiple scenarios -- it would be better to break this up into multiple tests which each validate a specific workflow or exception scenario.